diff -urN fbpanel-6.1/configure fbpanel-6.1-patched/configure
--- fbpanel-6.1/configure	2013-06-23 12:40:31.873471551 +1100
+++ fbpanel-6.1-patched/configure	2013-06-23 12:44:52.851950962 +1100
@@ -323,6 +323,10 @@
 add_var host  "stab: configure for arch" ""
 add_var target "stab: configure for arch" ""
 
+# more stabs
+add_var exec_prefix "stab: nothing" ""
+add_var sharedstatedir "stab: nothing" ""
+
 # N.B. order is important
 add_var prefix "install architecture-independent files" /usr
 add_var eprefix "install architecture-dependent files" '$prefix'
